DATA/production/configurations/2022/LHC22f/apass1/getTPCvdrift.C

```diff
@@ -53,6 +53,10 @@ float getTPCvdrift(int run, std::string_view ltrUrl = "http://alice-ccdb.cern.ch
   const auto ltrCalib = c.retrieveFromTFileAny<o2::tpc::LtrCalibData>(calibType.data(), metadata, minTime); /// timestamp in the run of interest
   const auto corr = ltrCalib->getDriftVCorrection();
   const float vcorr = defaultDriftV / corr;
+  if (ltrCalib->refVDrift != 0) {
+    printf("refVDrift different from zero: %f\n", ltrCalib->refVDrift);
+    vcorr = refVDrift / corr;
+  }
   printf("vdrift = %f\n", vcorr);
```

Comment:
Yes in spirit, but you cannot reassign the `const float vcorr`.

Reply:
Ah yes, sorry :-) I will fix it.

Reply:
Fixed.
Note, though, that I tried it for run 523148 from 22m, and it accessed the object in https://alice-ccdb.cern.ch/browse/TPC/Calib/LaserTracks/1660197371503, created on 3.08. Here the refVdrift is the same as the default. Does it mean that from some point in time on it is always set equal to the default? And it was zero only long ago?